src/roman_datamodels/maker_utils/_datamodels.py

```diff
@@ -0,0 +1,432 @@
+    ramp["border_ref_pix_bottom"] = u.Quantity(np.zeros((shape[0], 4, shape[2]), dtype=np.float32), u.DN, dtype=np.float32)
+
+    # and their dq arrays
+    ramp["dq_border_ref_pix_left"] = np.zeros((shape[1], 4), dtype=np.uint32)
+    ramp["dq_border_ref_pix_right"] = np.zeros((shape[1], 4), dtype=np.uint32)
+    ramp["dq_border_ref_pix_top"] = np.zeros((4, shape[2]), dtype=np.uint32)
+    ramp["dq_border_ref_pix_bottom"] = np.zeros((4, shape[2]), dtype=np.uint32)
+
+    # add amp 33 ref pixel array
+    ramp["amp33"] = u.Quantity(np.zeros((shape[0], shape[1], 128), dtype=np.uint16), u.DN, dtype=np.uint16)
+
+    ramp["data"] = u.Quantity(np.full(shape, 1.0, dtype=np.float32), u.DN, dtype=np.float32)
+    ramp["pixeldq"] = np.zeros(shape[1:], dtype=np.uint32)
+    ramp["groupdq"] = np.zeros(shape, dtype=np.uint8)
+    ramp["err"] = u.Quantity(np.zeros(shape, dtype=np.float32), u.DN, dtype=np.float32)
+
+    if filepath:
+        af = asdf.AsdfFile()
+        af.tree = {"roman": ramp}
+        af.write_to(filepath)
+    else:
+        return ramp
+
+
+def mk_ramp_fit_output(shape=(8, 4096, 4096), filepath=None):
+    """
+    Create a dummy Rampfit Output instance (or file) with arrays and valid values for attributes
+    required by the schema.
+
+    Parameters
+    ----------
+    shape
+        (optional) Shape of arrays in the model.
+
+    filepath
+        (optional) File name and path to write model to.
+
+    Returns
+    -------
+    roman_datamodels.stnode.RampFitOutput
+    """
+    meta = mk_common_meta()
+    rampfitoutput = stnode.RampFitOutput()
+    rampfitoutput["meta"] = meta
+
+    rampfitoutput["slope"] = u.Quantity(np.zeros(shape, dtype=np.float32), u.electron / u.s, dtype=np.float32)
+    rampfitoutput["sigslope"] = u.Quantity(np.zeros(shape, dtype=np.float32), u.electron / u.s, dtype=np.float32)
+    rampfitoutput["yint"] = u.Quantity(np.zeros(shape, dtype=np.float32), u.electron, dtype=np.float32)
+    rampfitoutput["sigyint"] = u.Quantity(np.zeros(shape, dtype=np.float32), u.electron, dtype=np.float32)
+    rampfitoutput["pedestal"] = u.Quantity(np.zeros(shape[1:], dtype=np.float32), u.electron, dtype=np.float32)
+    rampfitoutput["weights"] = np.zeros(shape, dtype=np.float32)
+    rampfitoutput["crmag"] = u.Quantity(np.zeros(shape, dtype=np.float32), u.electron, dtype=np.float32)
+    rampfitoutput["var_poisson"] = u.Quantity(np.zeros(shape, dtype=np.float32), u.electron**2 / u.s**2, dtype=np.float32)
+    rampfitoutput["var_rnoise"] = u.Quantity(np.zeros(shape, dtype=np.float32), u.electron**2 / u.s**2, dtype=np.float32)
+
+    if filepath:
+        af = asdf.AsdfFile()
+        af.tree = {"roman": rampfitoutput}
+        af.write_to(filepath)
+    else:
+        return rampfitoutput
+
+
+def mk_rampfitoutput(**kwargs):
+    warnings.warn("mk_rampfitoutput is deprecated. Use mk_rampfit_output instead.", DeprecationWarning)
+
+    return mk_ramp_fit_output(**kwargs)
```

Comment:
This method was a deviation from the naming convention which had a clear path to a proper name. I have left it in for now to not break anything, but it raises a `DeprecationWarning` now.